src/python/deepgnn/train_types.py

```diff
@@ -11,14 +11,12 @@ class TrainerType(Enum):
     * BASE: The most basic local trainer, used for simple experiments.
     * PS: Parameter server based distributed trainer, used for tensorflow models only.
     * HVD: Horovod based distributed trainer, supports both tensorflow and pytorch models.
-    * DDP: DistributedDataParallel based distributed trainer, used for pytorch models only.
     """
 
     BASE = "base"
     PS = "ps"
     MULTINODE = "multinode"
     HVD = "hvd"
-    DDP = "ddp"
```

Comment:
That's a breaking change. Why it is not mentioned in changelog?

Reply:
I still don't see it in changelog

Reply:
![image](https://user-images.githubusercontent.com/134559/221025069-3eca80e1-e1be-41e4-83f9-6103ae964e5f.png)

Reply:
Does this give enough info: - Remove our custom PyTorch trainers, use Ray instead.

Reply:
No, the changelog comment is about trainers and this is a const. The confusing part is hvd, the value is still present, but trainer was removed.